easybuild/easyconfigs/c/Cereal/Cereal-1.3.0.eb

```
@@ -16,15 +16,20 @@ used standalone."""
 
 toolchain = SYSTEM
 
-source_urls = ['https://github.com/USCiLab/cereal/archive/']
+github_account = 'USCiLab'
+source_urls = [GITHUB_LOWER_SOURCE]
 sources = ['v%(version)s.tar.gz']
 checksums = ['329ea3e3130b026c03a4acc50e168e7daff4e6e661bc6a7dfec0d77b570851d5']
-builddependencies = [('CMake', '3.12.1')]
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+
 configopts = '-DJUST_INSTALL_CEREAL=ON -DSKIP_PERFORMANCE_COMPARISON=ON '
 
 sanity_check_paths = {
     'files': ['include/cereal/cereal.hpp'],
-    'dirs': ['include', 'share'],
+    'dirs': ['include', 'lib/cmake'],
```

```suggestion
    'dirs': ['include', 'share/cmake/cereal'],
```

That is where that turns up for me.

---

(I just guessed it was going to be the same as for 1.3.2)